### ISSUE TITLE
Request: +Railroads

### ISSUE THREAD

**Issue by sys0pt** [orangeadam3/terra121#23]
Could you add Railroads like roads and pathways pls? :)

**Comment by orangeadam3** [comment]
Yeah, I'm just not sure how to automatically place rails nicely. It's probably possible I just haven't gotten around to it.

**Comment by sys0pt** [comment]
Instead of placing actual rail blocks its better 2 draw them, so if one is using Immersive Railroading for example, he can place the specific rails on the pre existing rail bed.
